cypress/e2e/tests/migration/task-manager/application_tasks_tab.test.ts

```diff
@@ -0,0 +1,74 @@
+
+/*
+Copyright © 2021 the Konveyor Contributors (https://konveyor.io/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+/// <reference types="cypress" />
+import {
+    login,
+    getRandomApplicationData,
+    sidedrawerTab,
+    deleteByList,
+    getRandomAnalysisData,
+} from "../../../../utils/utils";
+import { Application } from "../../../../e2e/models/migration/applicationinventory/application";
+import { Analysis } from "../../../models/migration/applicationinventory/analysis";
+   
+let applicationsList: Array<Analysis> = [];
```

Comment:
Since you are creating just one application you don't need the array.

You could just store the single app you have in the `application` variable and delete it in the `afer` hook with `application.delete();`